### PR TITLE
Add class structure to best practises

### PR DESCRIPTION
**Pull request by marijnschilling**
Now that our team is growing I think it's important to have a guideline for the way we order the elements in code. 

It's very important to be able to find your way quickly in classes to be able to work efficient as a team! 💪 👨‍👩‍👧‍👦:✨ 

Very much open to suggestions for switching the orders! I just suggested _some_ order here as a starting point!

### CODE REVIEW

File: BEST_PRACTICES.md [hunk 1]
```diff
@@ -57,6 +58,35 @@ When making backwards compatible changes, flag your old APIs as deprecated like
 
 When it comes to apps, patch releases are bug fixes, minor releases are small new features and major releases are re-designs or big features.
 
+## Class structure
+
+To make the code structured and to make it easy for yourself and other developers to find things in the code we have a guide for codestructure. It tells you where to put what and in what order.
+
+#### Order of views
+When declaring, laying-out or adding your views always keep the order of the views in the design. The order to follow is: from back till front and in reading order.
```
Comment (elland) [inline BEST_PRACTICES.md]: back `to` front? 👍

File: BEST_PRACTICES.md [hunk 2]
```diff
@@ -57,6 +58,35 @@ When making backwards compatible changes, flag your old APIs as deprecated like
...
+
+![Example of view order](https://raw.githubusercontent.com/bakkenbaeck/iOS-playbook/master/assets/view-order-example)
+
+So for example when implementing the view above the order would be:
```
Comment (elland) [inline BEST_PRACTICES.md]: Remove `So `. Repetitive.  Also commas. `For example, ` `the view above,`.

**Comment by marijnschilling** [comment]
i don't know how to add the image :(

File: BEST_PRACTICES.md [hunk 3]
```diff
@@ -57,6 +58,35 @@ When making backwards compatible changes, flag your old APIs as deprecated like
...
+Order of code (for every element counts: first place the open and underneath the private)
+0. protocols above the class declaration)
+1. enums
+2. static lets
+3. normal properties
+4. computed poperties
+5. observed properties
+6. lazy vars
+7. init
+8. view controller life cycle methods (in order how the are called)
```
Comment (elland) [inline BEST_PRACTICES.md]: Just life-cycle methods, could be a view, model, etc.

**Review by yuliaveres — APPROVED** [review]
Yes, I like it 👍  💯  🏖

**Review by roberthein — APPROVED** [review]
✨ Top!

**Comment by marijnschilling** [comment]
I made the structure guidelines looser. I feel that we as a team are pretty much on the same page on this point. So it's better to trust each others judgement when ordering code to some extend, then to compare every thing to a super detailed guideline.

**Pull request merged**